README.md

```diff
@@ -1,17 +1,54 @@
-# Software Engineering B4 Project Repo
+# PyramidQuest
+## What is PyramidQuest
+PyramidQuest is a game that aims to teach children educational knowledge in a playful way. As a character you are an archaeologist who is in a pyramid and is looking for a way up. This pyramid consists of two mazes. In the labyrinths he encounters obstacles such as: puzzles when opening doors, as well as different types of enemies and traps. There are 3 stars in each of the two levels that you can collect to get a better score. There is also a timer that the child can use to measure their time.     
 
-### Projektmanagement:
-* Manuel
-* Leon
+### Built With
 
-### Vertriebsmanagement:
-* Amier
-* Lucas
+* [Unity]
 
-### Techmanagement:
-* Baran
-* Hannah
+## Getting Started
+To get a local copy up and running follow these simple steps.
 
-### Qualitätmanagement:
-* Nam
-* Ikram
+### Prerequisites
+
+You need a Windows 10 operating system or newer.
+
+### Installation
+    
+ To start the Game, you need to download WinRaR. 
+* [WinRar](https://www.win-rar.com/predownload.html?&L=1)
+
+
+ Download the Game`[.rar]`
+*  [PyramidQuest.rar](https://github.com/B4-Group/swe_b4/releases/download/v1.3/PyramidQuest.v1.3.rar)
+1.  Open the folder PyrmaidQuest
+2.  Click on PyramidQuest
+3.  Click on `PyramidQuest.rar`
+
+The Game get started
+
+## Feature 
+
+See the [Lastenheft](https://drive.google.com/file/d/1Zo8HbJB4aVkVHRcmlVdS-qxpoCjMNVfY/view?usp=sharing) for a full list of proposed features.
```

Comment:
This Link won't work for anyone other than out team members.

Reply:
public?